portal-api/src/v1/validation/bond-rules/facility-stage-issued-rules/requested-cover-start-date.js

```diff
@@ -60,43 +60,44 @@ module.exports = (submittedValues, deal, errorList) => {
           };
         }
       }
-      if (coverDayValidation.error && requestedCoverStartDateDay) {
-        // error object does not exist if no errors in validation
-        newErrorList.requestedCoverStartDate = {
-          text: 'The day for the requested Cover Start Date must include 1 or 2 numbers',
-          order: orderNumber(newErrorList),
-        };
-      }
+    }
+    if (coverDayValidation.error && requestedCoverStartDateDay) {
+      // error object does not exist if no errors in validation
```

Comment:
I am not entirely sure what does this comment mean please?

Reply:
done